plugins/default/check_pod_disk_usage.sh

```diff
@@ -0,0 +1,106 @@
+################################
+# GLOBALS
+################################
+# Nagios exit codes
+NAGIOS_OK=0
+NAGIOS_WARN=1
+NAGIOS_CRIT=2
+NAGIOS_UNKNOWN=3
+ 
+################################
+# FUNCTIONS
+################################
+usage() {
+ echo "Usage: $0 -w <warn> -c <crit> [-m <usage|inodes>] [-h]"
+ echo " -w <warn> Warning threshold percentage (e.g. 80)"
+ echo " -c <crit> Critical threshold percentage (e.g. 90)"
+ echo " -m <mode> Check mode (usage or inodes)"
+ echo " -h        Show this message"
+ echo "Example: $0 -w 85 -c 95"
+}
+
+################################
+# ARGUMENT PROCESSING
+################################
+THRESH_WARN=
+THRESH_CRIT=
+CHECK_MODE=usage
+while getopts “hw:c:m:” OPTION
+do
+ case $OPTION in
+  h) usage; exit $NAGIOS_UNKNOWN ;;
+  w) THRESH_WARN=$OPTARG ;;
+  c) THRESH_CRIT=$OPTARG ;;
+  m) CHECK_MODE=$OPTARG ;;
+  ?) usage; exit $NAGIOS_UNKNOWN ;;
+ esac
+done
+ 
+if [[ -z $THRESH_WARN ]] || [[ -z $THRESH_CRIT ]]; then
+ usage; exit $NAGIOS_UNKNOWN
+fi
+
+TEMP_FILE=`mktemp`
+
+if [ "$CHECK_MODE" = "usage" ]; then
+	DF_CMD="df -x tmpfs"
+elif [ "$CHECK_MODE" = "inodes" ]; then
+	DF_CMD="df -i -x tmpfs"
+else
+	echo "Unknown check mode specified"
+	usage; exit $NAGIOS_UNKNOWN
+fi
+
+oc get pod -o json | jq -r '.items[] | select(.status.phase == "Running") | .metadata | [.namespace, .name] | @tsv' | sort | while IFS=; read podns; do
+	ns=`echo $podns | awk '{print $1}'`
+	pod=`echo $podns | awk '{print $2}'`
+	oc exec $pod -n $ns -- bash -c "$DF_CMD | grep -v Filesystem" | while read line; do
```

Comment:
We control what our mount points are on our components, so I wouldn't think we would ever allow that to be the case, but I see no harm in using tail to avoid the header line like you did in your python based PR.